test/types/index.ts

```diff
@@ -0,0 +1,26 @@
+import * as Fastify from 'fastify'
+import * as fastifyStatic from '../..'
+
+const options: fastifyStatic.FastifyStaticOptions = {
+  acceptRanges: true,
+  'cacheControl': true,
+  'decorateReply': true,
+  'dotfiles': true,
+  'etag': true,
+  'extensions': ['.js'],
+  'immutable': true,
+  'index': ['1'],
+  'lastModified': true,
+  'maxAge': '',
+  'prefix': '',
+  'root': '',
+  'schemaHide': true,
+  'serve': true,
+  'setHeaders': (res, pathName) => {
+    res.setHeader('test', pathName)
+  }
+}
+
+const app = Fastify()
+
+app.register<fastifyStatic.FastifyStaticOptions>(fastifyStatic, options)
```

Comment:
Can you add a test for sendFile as well?

Reply:
Added a test case.